scrapper.py

```diff
@@ -25,39 +36,159 @@ class Crawler:
+        for seed_url in self.seed_urls:
+            issue_page = requests.get(seed_url)
+            issue_page_bs = BeautifulSoup(issue_page.text, 'html.parser')
+            self._extract_url(issue_page_bs)
 
     def get_search_urls(self):
         """
         Returns seed_urls param
         """
-        pass
+        return self.seed_urls
+
+
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.url = article_url
+        self.article_id = article_id
+        self.article = Article(article_url, article_id)
+
+    def parse(self):
+        article_page = requests.get(self.url)
+        article_bs = BeautifulSoup(article_page.text, 'html.parser')
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        return self.article
+
+    def _fill_article_with_text(self, article_bs):
+        list_of_possible_l = ['Список литературы', 'ЛИТЕРАТУРА', 'Список использованных источников и литературы',
+                              'Источники']
+
+        class_of_link = article_bs.find('div', {'class': "articles-item-more"})
+        if class_of_link:
+            link_to_save = class_of_link.find('a')
+            link_to_pdf = 'https://periodical.pstgu.ru' + link_to_save['href']
+            final_page = requests.get(link_to_pdf)
+            final_page_bs = BeautifulSoup(final_page.text, 'html.parser')
+        else:
+            final_page_bs = article_bs
+        url_pattern = re.compile(r"var DEFAULT_URL = '(.*?)'")
+        final_link = final_page_bs.find(string=url_pattern)
+        final_link = 'https://periodical.pstgu.ru' + \
+                     str(final_link).replace('var DEFAULT_URL = ', '').replace("'", '').replace(';', '').strip()
+        raw_pdf = PDFRawFile(final_link, self.article_id)
+        raw_pdf.download()
+        text_from_pdf = raw_pdf.get_text()
+        for refs_maker in list_of_possible_l:
+            if refs_maker in text_from_pdf:
+                text_pdf_list = text_from_pdf.split(refs_maker)
+                del text_pdf_list[-1]
+                text_from_pdf = ' '.join(text_pdf_list)
+        self.article.text = text_from_pdf
+
+    def _fill_article_with_meta_information(self, article_bs):
+        issue_number = []
+        month_dict = {2: '03', 3: '06', 0: '09', 1: '12'}
+        try:
+            author = article_bs.find('p', {"class": "articles-item-author"}).find('b')
+            self.article.author = author.text
+        except AttributeError:
+            self.article.author = 'No data'
+        title = article_bs.find('h2', {"class": "articles-item-name"})
+        self.article.title = title.text
+        topics = article_bs.find('div', {"class": "articles"}).find_all('p')
+        try:
+            self.article.topics = topics[2].text
+        except IndexError:
+            self.article.topics = 'No data'
```

Comment:
that is not correct, consult the docs. I am sure you already know it thanks to tests